src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
+
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+
+  background-color: #f1f1f1;
+}
+
+.header {
+  box-sizing: border-box;
+  padding: 0 40px;
+  display: flex;
+  justify-content: space-between;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header_large {
+  /* max-width: 1200px; */
+  margin-bottom: 62px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px;
```

Comment:
no need margins,
just make for header paddings 0 50px

Reply:
each li has left and right margins 10 px. If i make paddings 0 50px nav will move to the left by extra 10px. So in this case I'll have to write extra pseudo and reset margin-right for "VIDEO"